flow.js

```diff
@@ -184,6 +189,9 @@ class Flow {
         }
         nextFlowStep = condition ? condition.nextStep : state.step;
       }
+      if(nextFlowStep == 'repair-description' && repairProblemChanged){
```

Comment:
Please move the setting of `repairProblemChanged` to here which is closer to its usage to only set it when needed and make it's usage easier to understand.

Reply:
The reason we have it at the top is because were soing the check before the state is updated which happens on the first line of the function